modules/zero-trust-manager-about-spire.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * security/zero_trust_workload_identity_manageer/zero-trust-manager-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="zero-trust-manager-about-spire_{context}"]
+= SPIRE server
+
+
+A SPIRE server is composed of one or more SPIRE agents. The server manages and issues SPIFFE identities within a trust domain. It stores registration entries (selectors that determine under what conditions a SPIFFE ID should be issued) and signing keys. The server also performs node and workload attestation to verify the identity of the node or workload requesting a {svid-full} based on predefined conditions. For more information, see link:https://spiffe.io/docs/latest/spire-about/spire-concepts/#spire-architecture-and-components[SPIRE architecture and components].
```

Comment:
SPIRE server is not composed of SPIRE agents but has a set of plugins such as the OIDC plugin. See  https://spiffe.io/docs/latest/spire-about/spire-concepts/#all-about-the-server
Also, Only SPIRE agents are involved in Workload attestation https://spiffe.io/docs/latest/spire-about/spire-concepts/#workload-attestation